prow/gerrit/adapter/adapter.go

```diff
@@ -148,7 +148,8 @@ func (c *Controller) SaveLastSync(lastSync time.Time) error {
 // Sync looks for newly made gerrit changes
 // and creates prowjobs according to presubmit specs
 func (c *Controller) Sync() error {
-	syncTime := time.Now()
+	// gerrit timestamp only has second precision
+	syncTime := time.Now().Truncate(time.Second)
```

Comment:
we should probably put a TODO here? and maybe also truncate the received timestamp?

Reply:
(discussed offline, we're already truncating received via the parse format)

Reply:
added TODO and opened https://github.com/kubernetes/test-infra/issues/9545